main.go

```diff
@@ -207,6 +209,28 @@ func populateArgs(c *cli.Context) (
 	return
 }
 
+func callListRecursive(mountPoint string) (err error) {
+	logger.Debugf("Started recursive metadata-prefetch of directory: \"%s\" ...", mountPoint)
+	numItems := 0
+	err = filepath.WalkDir(mountPoint, func(path string, d fs.DirEntry, err error) error {
+		//logger.Infof("Walked path:%s, d=%s, isDir=%v", path, d.Name(), d.IsDir())
+		if err != nil {
+			if d == nil {
+				return fmt.Errorf("got error walking: path=\"%s\" does not exist, error = %w", path, err)
+			} else {
+				return fmt.Errorf("got error walking: path=\"%s\", dentry=\"%s\", isDir=%v, error = %w", path, d.Name(), d.IsDir(), err)
+			}
+		}
+
+		numItems++
+		return err
+	})
+
+	logger.Debugf("... Completed recursive metadata-prefetch of directory: \"%s\". Number of items discovered: %v", mountPoint, numItems)
+
+	return
```

Comment:
Naked returns harm readability. Please fix.

Reply:
I have seen this as norm throughout the GCSFuse code. But I'll change it here to make it more readable.